runtime/stdlib/flow.go

```diff
@@ -209,6 +209,21 @@ var AccountEventContractParameter = sema.Parameter{
 	TypeAnnotation: sema.StringTypeAnnotation,
 }
 
+var AccountEventHashAlgorithmParameter = sema.Parameter{
+	Identifier:     "hashAlgorithm",
+	TypeAnnotation: sema.UInt8TypeAnnotation,
+}
+
+var AccountEventKeyWeightParameter = sema.Parameter{
+	Identifier:     "weight",
+	TypeAnnotation: sema.IntTypeAnnotation,
```

Comment:
It's unfortunate that we used `UFix64` for the weight in the Cadence API, in `AccountKey` and in `keys.add`, but under the hood it's just an `int`.

Still, to maybe be consistent, use `UFix64` here.